h2o-py/tests/testdir_misc/pyunit_pubdev_6672_2D_pdp_fix.py

```diff
@@ -0,0 +1,23 @@
+import sys
+import os
+sys.path.insert(1,"../../")
+import h2o
+import tempfile
+from tests import pyunit_utils
+from h2o.estimators.glm import H2OGeneralizedLinearEstimator
+
+'''
+Copied example from Honza to fix bug in 2D pdp.  Basically, when no figure is plotted, this bug will thrown an error.
+Hence, this code just needs to run to completion.  No assertion check is necessary for this test.
+'''
+def partial_plot_test_with_user_splits():
+    train = h2o.import_file(pyunit_utils.locate('smalldata/flow_examples/abalone.csv.gz'))
+    model = H2OGeneralizedLinearEstimator(training_frame=train)
+    model.train(y="C9")
+    model.partial_plot(train, col_pairs_2dpdp=[["C1", "C2"]], save_to_file="pdp.png")
```

Comment:
should the test cleanup the created file? do we even need to save_to_file to reproduce this?

Reply:
In this case, there is no file saved since no plot is generated.  No need to do clean up in this case.

Reply:
We do not need to specify the save option to reproduce the error.

Reply:
thats right, im which case we should remove it from the test to avoid confusion